src/components/editMeasure/details/TransmissionFormat/TransmissionFormat.tsx

```diff
@@ -84,12 +84,7 @@ const TransmissionFormat = (props: TransmissionFormatProps) => {
     initialValues: { ...INITIAL_VALUES },
     enableReinitialize: true,
     validationSchema: Yup.object().shape({
-      transmissionFormat: Yup.string()
-        .notRequired()
-        .max(
-          250,
-          "Measure Transmission Format cannot be more than 250 characters."
-        ),
+      transmissionFormat: Yup.string(),
```

Comment:
Sorry, I should have been more clear. I mean whole validation schema- do we really need? we are not enforcing anything apart from value being string and anything can be included in string. I was wondering if we could just remove validation schema altogether.